packages/wrangler/src/versions/view.ts

```diff
@@ -1,141 +1,141 @@
 import { logRaw } from "@cloudflare/cli";
+import { createCommand } from "../core/create-command";
 import { UserError } from "../errors";
 import * as metrics from "../metrics";
 import { printWranglerBanner } from "../update-check";
 import { requireAuth } from "../user";
 import formatLabelledValues from "../utils/render-labelled-values";
 import { fetchVersion } from "./api";
-import { getConfig, getVersionSource } from "./list";
+import { getVersionSource } from "./list";
 import type { WorkerMetadataBinding } from "../deployment-bundle/create-worker-upload-form";
-import type {
-	CommonYargsArgv,
-	StrictYargsOptionsToInterface,
-} from "../yargs-types";
 
 const BLANK_INPUT = "-"; // To be used where optional user-input is displayed and the value is nullish
 
-export type VersionsViewArgs = StrictYargsOptionsToInterface<
-	typeof versionsViewOptions
->;
-
-export function versionsViewOptions(yargs: CommonYargsArgv) {
-	return yargs
-		.positional("version-id", {
+export const versionsViewCommand = createCommand({
+	metadata: {
+		description: "View the details of a specific version of your Worker",
+		owner: "Workers: Authoring and Testing",
+		status: "open-beta",
```

Comment:
I think this is now GA: https://blog.cloudflare.com/builder-day-2024-announcements/#gradual-deployments